src/GetArticle.jsx

```diff
@@ -13,23 +17,58 @@ export default function GetArticle() {
       })
       .then((result) => {
         setArticleItem(result.article[0]);
+        setIsLoading(false);
       });
   }, []);
 
-  return (
-    <div key={articleItem.article_id}>
-      <h1>{articleItem.title}</h1>
-      <div className="article-header-holder">
-        <p>Article by: {articleItem.author}</p>
-        <p>
-          Votes: {articleItem.votes}
-          <button>UPVOTE TO GO HERE</button>
-        </p>
+  function handleUpvote(event, increment = 1) {
+    event.preventDefault();
+    const updateObject = {};
+    updateObject.inc_votes = increment;
+    axios
```

Comment:
It would be beneficial to abstract your api calls out to an api.js file and call them where needed.